includes/results.inc

```diff
@@ -1097,8 +1097,11 @@ class IslandoraSolrFacets {
       $buckets[] = $link;
     }
 
+    drupal_alter('islandora_solr_facet_variables', $buckets, $facet_field);
+dd($facet_field);
```

Comment:
Not really related to the this PR, but just seeing this: I'm... somewhat leery about what sniffs Travis is running now, 'cause it's not flagging the debug call here?